docker-compose.yml

```diff
@@ -82,6 +82,7 @@ services:
       - api/env.docker
     stdin_open: true
     tty: true
+    user: ${DOCKER_USER_ID}:${DOCKER_GROUP_ID}
```

Comment:
I recommend setting defaults for this to avoid these warnings. It seems inevitable  to me that we will occasionally invoke `docker compose` outside of a `just` recipe. 

<img width="677" alt="Screenshot 2022-08-17 at 5 52 40 PM" src="https://user-images.githubusercontent.com/16580576/185117821-0d553c6a-6f8b-41e2-adc6-5967f507a3a2.png">

Reply:
> It seems inevitable to me that we will occasionally invoke docker compose outside of a just recipe

Under what circumstances would one do this? If there aren't concrete use cases then I don't think we should worry about supporting them. In fact, supporting them could get us into trouble as we would have to continue to monitor those workflows.

Given how trivial it is to add new recipes to the justfile I think we should just expect people to always go through `just`... we have enough specific configuration stuff as it is in there that it's non-trivial to run a `docker compose` command that will actually work outside very specific circumstances.

Reply:
I regularly run `docker-compose ps` to see the status of services, `docker-compose restart` to restart some services etc. Going through just for every use case of `docker-compose` is not feasible.

Reply:
We can fairly easily create a `dc` recipe for this:

```make
dc *args:
    docker-compose {{ DOCKER_FILE }} *args
```

We don't have to re-write every command. It would simplify our justfile as well. All the `docker-compose {{ DOCKER_FILE }}` stuff can be replaced with `just dc`.

Reply:
PR for the above suggestion: https://github.com/WordPress/openverse-api/pull/886